src-v2/Judge/NTS.Judge.Blazor/Pages/Setup/Competitions/CompetitionUpdate.razor

```diff
@@ -11,7 +11,7 @@
 @inherits NotComponent
 
 <MudPaper Elevation="0" Class="pl-6 pr-6">
-    <NotH2>@Localizer.Get("Update Competition")</NotH2>
+    <NotH2>@Localizer.Get("Edit Competition")</NotH2>
```

Comment:
Because from user POV they are about to edit the records, then after editing is completed the Update action is called which isn't performed by the user. My idea is to differentiate these and place the 'responsibility' for editing on the user and the responsibility for updating on the system.

Reply:
While probably more semantically correct, I don't find it a meaningful distinction. Instead I prefer to stick to simpler scenario where we have one term and we share terminology with the user, rather then diverge from it. It would be different if from a software standpoint we need to segregate different process stages and call one Edit and another Update, but I don't think that's necessary and desirable here.

Reply:
It's not that one way is more semantically correct, but from UX perspective the application should be intuitive to stimulate people to use it. UI should hint to the expected behavior.

 For this reason I haven't changed the labels of the Form buttons as the difference in software behavior is:
- Edit buttons open the *UpdateForm* 
- Update buttons send the edited data for persistence

In that sense I find that from software standpoint we already perform different actions with the different buttons and we call the entire process **Update** although it implicitly include Editing and it is up to the user to initiate the actual update or cancel the changes.

I just think having the same text will be detrimental from UX point of view, however I don't mean to argue as the final decision is yours.

Reply:
I see your point, but honestly I don't think people would make that distinction. I know I didn't. Also not entirely sure it's correct: [quick google search](https://www.google.com/search?q=what%27s+the+difference+between+edit+and+update&rlz=1C1YTUH_enBG1091BG1091&oq=what%27s+the+difference+between+edit+and+update&gs_lcrp=EgZjaHJvbWUyBggAEEUYOTIHCAEQABiABDIHCAIQABiABDIHCAMQABiABDIHCAQQABiABDIHCAUQABiABDIHCAYQABiABDIHCAcQABiABDIHCAgQABiABDIHCAkQABiABNIBCDUyNDdqMGo3qAIAsAIA&sourceid=chrome&ie=UTF-8). 

I think there are better ways to spend our time. Lets keep Update everywhere and for the future lets discuss things that fall outside of the scope of the issue, before implementing them and then having to revert back later.